Dockerfile

```diff
@@ -582,8 +582,8 @@ ENV PATH=$PATH:/opt/ort/bin
 USER $USER
 WORKDIR $HOME
 
-# Ensure that the ORT data directory exists to be able to mount the config into it with correct permissions.
-RUN mkdir -p "$HOME/.ort"
+# Ensure that data directories exist to be able to mount volumes from the host into them with correct permissions.
```

Comment:
Nit: Maybe just "that these directories" and scratch "into them" (same below).

Reply:
I somehow wanted to express that this mostly is about existing data on the host being made available in the container, and that there is only a problem if you mount "into" a nested path inside the container where any intermediate path is not existing in the container.

Reply:
I found that it sounds weird but it's just a nit anyway.